### ISSUE TITLE
TaxjarError: undefined - undefined

### ISSUE THREAD

**Issue by ruslan4k** [taxjar/taxjar-node#46]
We just started to use TaxJar trial.

On a local machine, all went fine, it’s working. But when we deployed it to GCP k8s, taxjar started throwing a strange error

No details at all.

```
{
detail: undefined, 
error: undefined, 
message: undefined - undefined, 
name: TaxjarError, 
stack: 
TaxjarError: undefined - undefined
    at proxyError (/home/node/app/node_modules/taxjar/dist/util/request.js:6:11)
, 
status: undefined
}
```
We use only **.taxForOrder** method.

package version **"taxjar": "3.1.0"**

**Comment by codymullins** [comment]
Hi @ruslan4k, thanks for reaching out. Just sent you an email to your publicly listed email address for some more information to help troubleshoot this issue.

**Comment by codymullins** [comment]
@ruslan4k - I was able to reproduce the same error by passing in an invalid `apiUrl`:

```
const client = new Taxjar({
    apiKey: 'MY_KEY',
    apiUrl: 'sdfsdf'
});
```

Can you confirm that your environment variable / configuration for the API URL is correct? If this isn't the issue, it would be helpful if you could send a snippet of your code and the values you are passing with any sensitive information scrubbed out.

**Comment by ruslan4k** [comment]
@codymullins Thanks for your help

We don't modify API URL in the config

this is the code snippet
```
const Taxjar = require('taxjar');

const taxjar = new Taxjar({
    apiKey: process.env.TAXJAR_API_KEY,
});

const { tax } = await taxjar.taxForOrder({
    from_country: country,
    from_zip: zip,
    from_state: state,
    from_city: city,
    from_street: street,
    to_country: countryCode,
    to_zip: address_zip,
    to_city: address_city,
    to_state: stateCode,
    to_street: address_line1,
    shipping: 0,
    line_items: [
        {
            quantity: 1,
            unit_price: totalAmount,
            product_tax_code: '30070',
        },
    ],
});
```

**Comment by ScottRudiger** [comment]
@ruslan4k An issue with masking non-TaxJar errors (`RequestError`, etc.) was resolved with the release of `v3.1.1`.

If you're still facing this issue, you can try upgrading to the newest version. This should give us some more details on the error you were getting. Let us know if we can help further either here or via support@taxjar.com. Thanks!
